src/prototypes.h

```diff
@@ -279,9 +279,9 @@ bool unc_homedir(std::string &home);
  * @return the next tabstop column
  */
 static_inline
-size_t calc_next_tab_column(int col, int tabsize)
+size_t calc_next_tab_column(size_t col, size_t tabsize)
```

Comment:
This function is a bit dangerous.

What should happen when:
* `tabsize` is 0? Right now it just explodes.
* (`cpd.frag_cols - 1`) is bigger than `col`? Here it will underflow the size_t.